frontend/src/data/query/MetricQuery.ts

```diff
@@ -1,19 +1,31 @@
-import { Breakdowns, BreakdownVar } from "./Breakdowns";
+import { Breakdowns, BreakdownVar, TimeView } from "./Breakdowns";
 import { Row, FieldRange } from "../utils/DatasetTypes";
 import { MetricId } from "../config/MetricConfig";
 import { DemographicGroup } from "../utils/Constants";
 
 export class MetricQuery {
   readonly metricIds: MetricId[];
   readonly breakdowns: Breakdowns;
+  readonly timeView: TimeView;
 
-  constructor(metricIds: MetricId | MetricId[], breakdowns: Breakdowns) {
+  constructor(
+    metricIds: MetricId | MetricId[],
+    breakdowns: Breakdowns,
+    timeView?: TimeView
+  ) {
     this.metricIds = [metricIds].flat();
     this.breakdowns = breakdowns;
+    this.timeView = timeView || "cross_sectional";
   }
 
   getUniqueKey(): string {
-    return this.metricIds.join(",") + ":____:" + this.breakdowns.getUniqueKey();
+    return (
+      this.metricIds.join(",") +
+      ":____:" +
+      this.breakdowns.getUniqueKey() +
+      ":____:" +
+      this.timeView
```

Comment:
Is there a reason for concatenating the strings with `+` rather than using template literals?